script/download-node.js

```diff
@@ -110,7 +115,8 @@ var downloadNode = function(version, done) {
   }
 };
 
-downloadNode('v0.10.35', function(error) {
+var apmNodeVersion = process.env.APM_NODE_VERSION || '0.10.35';
```

Comment:
> @zcbenz so is node 0.10 unable to build native modules for Electron 0.30? Should we just start shipping 0.12 instead of 0.10 then?

Building native modules doesn't rely on the version of Node, both 0.10 and 0.12 are fine.